gitutil/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/fluxcd/pkg/gitutil
+
+go 1.14
```

Comment:
```suggestion
go 1.15
```

We use 1.15 in git, runtime, meta, etc

Reply:
Ah yes, it picked up my local go version which I haven't bothered to update :-S